relay-server/src/services/store.rs

```diff
@@ -299,7 +299,13 @@ impl StoreService {
                     self.produce_span(scoping, start_time, event_id, retention, item)?
                 }
                 other => {
-                    relay_log::error!("StoreService received unexpected item type: {other}");
+                    let event_type = event_item.as_ref().map(|item| item.ty());
```

Comment:
Not sure how `event_type` is displayed, is it more readable with `.unwrap_or("null")`?